solr/solr-ref-guide/modules/deployment-guide/pages/system-requirements.adoc

```diff
@@ -81,15 +82,6 @@ In addition, some organizations also maintain their own test infrastructure and
 Our continuous testing is against the two code lines under active development, Solr 9x and the future Solr 10.0:
 
 * Solr 9.x is the current stable release line and will have "point releases", i.e., 9.1, 9.2, etc., until Solr 10.0 is released.
-** Solr 9.x is currently tested against Java 11, 17 and 18-prerelease.
+** Solr 9.x is continuously tested against Java 11, 17, 21 and also newer versions.
```

Comment:
Can you point to evidence of this?  I only see Java 11.  Back when Lucene & Solr were combined, it was more.

Reply:
The "continuously tested" text could be a link out to Jenkins

Reply:
Yes its continuously tested on Policeman Jenkins

Reply:
https://jenkins.thetaphi.de/view/Solr/